.github/workflows/build.yml

```diff
@@ -58,7 +60,7 @@ jobs:
     - name: Setup Python
       uses: actions/setup-python@v5
       with:
-        python-version: '3.13'
+        python-version: '3.10'
```

Comment:
why the downgrade?

Reply:
I was not working with 3.13. 3.10 is working. See https://github.com/sphinx-contrib/multiversion/issues/170

Reply:
Thanks for the backlink. LGTM for now.